tests/rules/no-multiple-actions-in-effects.test.ts

```diff
@@ -8,200 +8,171 @@ import { ruleTester } from '../utils'
 
 ruleTester().run(path.parse(__filename).name, rule, {
   valid: [
-    {
-      code: `
-        export class Effects {
-          seven$ = createEffect(() =>
-            this.actions$.pipe(map(() => foo()))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          five$ = createEffect(() =>
-            this.actions$.pipe(switchMap(() => {
-              return of(foo())
-            }))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          thirteen$ = createEffect(() =>
-            this.actions$.pipe(mapTo(foo()))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          eleven$ = createEffect(() =>
-            this.actions$.pipe(switchMapTop([foo()]))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          twelve$ = createEffect(() =>
-            this.actions$.pipe(aconcatMapTo([bar()]))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          nine$ = createEffect(() =>
-            this.actions$.pipe(mergeMapTo(foo()))
-          )
-        }`,
-      parserOptions: {
-        project: './tsconfig.tests.eslint.json',
-      },
-      filename: 'test.ts',
-    },
-    {
-      code: `
-        export class Effects {
-          six$ = createEffect(() =>
-            this.actions$.pipe(
-              exhaustMap(() => {
-                return of({}).pipe(
-                  map(response => foo()),
-                  catchError(() => of(bar()))
-                );
-              })
-            )
+    `
+      @Injectable()
+      export class Effects {
+        effectOK$ = createEffect(() =>
+          this.actions$.pipe(map(() => foo()))
+        )
+      }`,
+    `
+      @Injectable()
+      export class Effects {
+        effectOK1$ = createEffect(() =>
+          this.actions$.pipe(switchMap(() => {
+            return of(foo())
+          }))
+        )
+      }`,
+    `
+      const action = () => foo()
+      @Injectable()
+      export class Effects {
+        effectOK2$ = createEffect(() =>
+          this.actions$.pipe(mapTo(action()))
+        )
+      }`,
+    `
+      @Injectable()
+      export class Effects {
+        effectOK3$ = createEffect(() =>
+          this.actions$.pipe(switchMapTop([foo()]))
```

Comment:
Note that these tests will fail every applying this change

```suggestion
          this.actions$.pipe(switchMapTo([foo()]))
```